home/templates/reports/page_view_report.html

```diff
@@ -1,108 +1,43 @@
 {% extends 'wagtailadmin/reports/base_page_report.html' %}
-{% load i18n wagtailadmin_tags %}
 {{ page.extra_scripts|safe }}
 {% block titletag %} Page views {% endblock %}
 
-{% block content %}
-
-    <header role="banner">
-        <div class="row">
-            <div class="left">
-                <div class="col">
-                    <h1>
-                        {% if header_icon %}{% icon name=header_icon %}{% endif %}
-                        {{ title }}{% if subtitle %} <span>{{ subtitle }}</span>{% endif %}
-                    </h1>
-                </div>
-            </div>
-            <div class="right report__actions">
-                {% block actions %}
-                    {% if view.list_export %}
-                        <div class="dropdown dropdown-button match-width">
-                            <a href="{{ view.xlsx_export_url }}" class="button bicolor button--icon">{% icon name="download" wrapped=1 %}{% trans 'Download XLSX' %}</a>
-                            <div class="dropdown-toggle">{% icon name="arrow-down" %}</div>
-                            <ul>
-                                <li><a  class="button bicolor button--icon" href="{{ view.csv_export_url }}">{% icon name="download" wrapped=1 %}{% trans 'Download CSV' %}</a></li>
-                            </ul>
-                        </div>
-                    {% endif %}
-                {% endblock %}
-            </div>
-        </div>
-    </header>
```

Comment:
I'm not sure why we're removing this header, I don't see it being put anywhere else. But I don't know these templates well enough to know how everything links together, just wanted to raise it in case we're losing something here.

Reply:
the header is defined in Wagtail's other templates, we are inheriting from those templates and only overriding block listing. They have done a major UI overhaul in this release